local-tests/setup/tinny-person.ts

```diff
@@ -49,6 +49,10 @@ export class TinnyPerson {
     this.wallet = new ethers.Wallet(privateKey, this.provider);
   }
 
+  async getAuthMethodId(): Promise<string> {
+    return await EthWalletProvider.authMethodId(this.authMethod);
+  }
+
```

Comment:
We could, however modern JS handles this correctly without the extra microtask of unwrapping&wrapping the value in a promise. And also this keeps the stacktrace which, in my opinion, is more than enough to justify that cost, if it were still a thing

Even more, the [eslint rule to prevent this](https://eslint.org/docs/latest/rules/no-return-await) was deprecated because of that

Made an example [here](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBANiA5jAvDUkRwKYDoGIDcAUMRtDAO4CGAlrGgBQC2EAXDGAK7MBG2AJwCUqAHydslGAAUBIZrQjYAPADcQtACajGjAdghZV2ESnFKoAFVrNsILlF2nx+w3GOMhAGhishQkmJqCABPMGAYADMucKhacBgoAAs5SgBRAA9gbAAHOPAAQUioQQKaek8YAG9iGBhqcthGxgBGAAY2gNrElJApMEkYNIE5AUYAcksACQAlAHkFxaWlgHU1lYBCLY3xroBfUmCwiOjY+LAomOB8sEsk6igAYWo4OAgCsALQ8LvU2jBEAAxK43RiNWTyRTYDi8EBYbDUMAiGp1WiRGBguhQCEKJTI7p1fRQLgCC4NLE9VKZbJ5c5FEoCMpYzwkOp7GDYN7YaoEmBEkkXZJUrK5G700rNLpskgHILfE4g86+OhgSoo+BICYzGJgf7IZgq3asxICEI8up1cn0S5ncB3B7PV7vT7y359PXA22qqACLgmY3s4APYBJDEmc0W8iwQSjVAc+oQGAxADWYD6ZMTw1GxrqBAmTPoepgOTkuOwPhjIAEMEUbCNvLzlYEuGg1GAyalMFl3R9ZvVlsaNuu53tTxebw+X2ObsoHsV4EYkReeID6GDocY4f76HAFCbce5wSTYFT6YTQxGVZzGsQEwAciBYFa4gDi6WoRXL9Xa-WLTfN1+LZQG2HYBsQsoGv8ni4EGUAhow5DwrgTZdEAA) 😃